web/src/main/webapp/my-app/layout/spec/layout_controller_spec.js

```diff
@@ -95,10 +95,10 @@ define(['angular-mocks', 'portal', 'my-app'], function() {
 
       it('should set layoutEmpty to true after return empty layout',
         function() {
-          httpBackend.whenGET(groupURL).respond([]);
```

Comment:
8d06bdcbf8af3c94408c36b9bc814969999c1dd5 is a whitespace-only commit. Consider a [commit message more like `style: normalize indent in layoutEmpty true test`](https://github.com/angular/angular/blob/22b96b9/CONTRIBUTING.md#type), rather than `refactor: test changes`.

Conventional Commits are nothing like sufficient for good software, but they do serve a purpose -- I had to remember to put a `w=1` on the end of the URL to figure out that this commit doesn't include changes to the test.